docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to create a project that you can build and run remotely. 
```

Comment:
Misleading - it is not _when_ you configure, it is an action you can perform only after you have configured Codewind to be used remotely.  (Bear in mind the style rules were relaxed around introducing the pre-requisites or next steps in the instruction flows. This is to prevent any misleading sentences that can arise from having to adhere to perfect tense.)

E.g. 
_Please ensure you have completed the prerequisites <link>
Now that you have configured Codewind to be used remotely, you are ready to create a project that you can develop locally but build and run remotely.